bindings/time-profiler.cc

```diff
@@ -70,7 +70,7 @@ NAN_METHOD(StartProfiling) {
 
   // Sample counts and timestamps are not used, so we do not need to record
   // samples.
-  cpuProfiler->StartProfiling(name, false);
+  cpuProfiler->StartProfiling(name);
```

Comment:
Why did you drop the "false" here? The comment to this line is I think about using the "false" here, we added it to avoid recording the timed sample information to reduce the profile size, what's the intention here?

Reply:
That's a mistake -- I'm guessing I merged badly at some point.
Thanks for catching this!